### PR TITLE
OKTA-600286 Add Org AS note to API service integrations

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add Org AS restriction note to API service integrations based on feedback from dev.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-600286](https://oktainc.atlassian.net/browse/OKTA-600286)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Build, test, and submit an API service integration to the Okta Integration Netwo
 A service-to-service app where a backend service or a daemon calls [Okta management APIs](/docs/reference/core-okta-api/) for a tenant (Okta org) can be published in the Okta Integration Network (OIN) as an API service integration.
 API service integrations access Okta APIs using the OAuth 2.0 [Client Credentials flow](/docs/concepts/oauth-openid/#client-credentials-flow), where access isn't associated with a user and you can restrict resources with scoped access tokens. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains.
 
-> **Note:** Currently, Okta only supports the OAuth APIs listed in [OAuth 2.0 Scopes for Okta Admin Management](https://developer.okta.com/docs/api/oauth2/#okta-admin-management).
+> **Note:** API service integrations can only use the [Org authorization server](/docs/concepts/auth-servers/#org-authorization-server) (Org AS) that is build-in with each Okta org. The Org AS supports the OAuth APIs listed in [OAuth 2.0 Scopes for Okta Admin Management](https://developer.okta.com/docs/api/oauth2/#okta-admin-management).
```
Comment (ramyamudduluri-okta) [inline packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md:28]: built-in instead of build-in

**Review by ramyamudduluri-okta — COMMENTED** [review]
Left a small nit, LGTM otherwise.

File: packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md:28]: sorry about that typo. will fix it now

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md](https://github.com/okta/okta-developer-docs/blob/20af65c89094b09b315f4fddd41ad80e9b6d0164/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-api-integration%2Fmain%2Findex.md) | [76](https://okta.acrolinx.cloud/api/v1/checking/scorecards/53c82cce-d3ed-4df6-802c-99115969522f) | 93  | 73  | 53  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/20af65c89094b09b315f4fddd41ad80e9b6d0164/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [74](https://okta.acrolinx.cloud/api/v1/checking/scorecards/eed3b8cb-9d12-4529-837e-037c2f9405e4) | 61  | 75  | 66  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.02ead829-20ea-4f8c-a1a1-aa0520d0a64e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by ramyamudduluri-okta — APPROVED** [review]
🚢

**Review by danielmaharry-okta — APPROVED** [review]

**Comment by tovamarkowitz-okta** [comment]
lgtm

**Pull request merged**